go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/rook/rook
 
-go 1.24.3
+go 1.24.10
```

Comment:
Why aren't we using toolchain in rook?
We still don't have 1.24.10 in d/s builds

Reply:
I'll fix this today in upstream was occupied with 4.21 items